### ISSUE TITLE
Create DB Schema for the api

### ISSUE THREAD

**Issue by Manpreet-Singh001** [CMPUT404W23-bigger-yoshi/CMPUT404-project-socialdistribution#44]
We need to create/discuss a schema for the api that we are gonna use. @shehraj123 I have assigned this issue to you. Please post any updates or your schema under this issue.

**Comment by shehraj123** [comment]
![ER drawio](https://user-images.githubusercontent.com/69915903/216850667-cebeef3f-aee0-4baa-9742-6f608daee69c.png)

**Comment by shehraj123** [comment]
Revised
![ER drawio](https://user-images.githubusercontent.com/69915903/216851953-42fff374-2a23-47aa-88f0-1c6a316dd840.png)

**Comment by Manpreet-Singh001** [comment]
```
post:
authorid post_id  meantfor:		
1 	   yyyy   manpreet
2 	   yyy    m
2 	   yyy    n
2 	   yyy    p
3 	   yyy    x1
3 	   yyy    y1
3 	   yyy    z1
3 	   yyy    m1
3 	   yyy    n1
3 	   yyy    p1

public to frens:
Delete for auth x -> meant_for not in (frens)

heavypost:
metadata
```

**Comment by Mattwmaster58** [comment]
closed by #60
